pkg/operator/ceph/nfs/controller.go

```diff
@@ -252,32 +252,33 @@ func (r *ReconcileCephNFS) reconcileCreateCephNFS(cephNFS *cephv1.CephNFS) (reco
 	if err != nil {
 		if kerrors.IsNotFound(err) {
 			logger.Infof("creating ceph nfs %q", cephNFS.Name)
-			err := r.upCephNFS(cephNFS, 0)
+			err := r.upCephNFS(cephNFS)
 			if err != nil {
 				return reconcile.Result{}, errors.Wrapf(err, "failed to create ceph nfs %q", cephNFS.Name)
 			}
 		} else {
 			return reconcile.Result{}, errors.Wrap(err, "failed to list ceph nfs deployments")
 		}
 	} else {
-		// Scale up case (CR value cephNFS.Spec.Server.Active changed)
 		nfsServerListNum := len(deployments.Items)
 		if nfsServerListNum < cephNFS.Spec.Server.Active {
```

Comment:
Why not remove this `if` statement since it does nothing now?

Reply:
I thought you were talking about something else when I realized it's not necessary to have so many `else` statements in the code if we return success after creating a new NFS cluster.